src/AzureAuth/Commands/Ado/CommandPat.cs

```diff
@@ -168,15 +174,22 @@ private bool ValidOptions(ILogger logger)
             }
             else
             {
-                var invalidScopes = AdoPat.Scopes.Validate(this.Scopes);
-                if (!invalidScopes.IsEmpty)
+                if(NoScopeValidation)
+                {
+                    logger.LogWarning($"Received {NoValidationFlag} flag. AzureAuth will skip validation of scopes.");
+                }
+                else
                 {
-                    foreach (var scope in invalidScopes)
+                    var invalidScopes = AdoPat.Scopes.Validate(this.Scopes);
+                    if (!invalidScopes.IsEmpty)
                     {
-                        logger.LogError($"{scope} is not a valid Azure DevOps PAT scope.");
+                        foreach (var scope in invalidScopes)
+                        {
+                            logger.LogError($"{scope} is not a valid Azure DevOps PAT scope.");
+                        }
+                        logger.LogError($"Consult {AdoPat.Constants.PatListURL} for a list of valid scopes.");

```

Comment:
We use the word `consult` but I guess all we recommend is that this link is your source of truth and if you still want to use custom scope then just use it with the `--no-validate` flag?

Reply:
Updated to `known/unknown`